Routines/Camera.py

```diff
@@ -0,0 +1,24 @@
+import picamera
+import time
+import os
+import threading
+data_files = 0
+camera = picamera.PiCamera()
+alert = 0
+lock = threading.Condition()
+class Camera(threading.Thread()):
+    #inittializing thread
+    def __init__(self, thread_name, thread_ID):
+        threading.Thread.__init__(self)
+        camera.vflip = True
+        data_files = 0
+        for files in os.walk("Data"):
+            data_files += 1
+    #run thread and record
+    def run(self):
+        global alert
+        footage_name = "Data/Footage" + (data_files+1)
```

Comment:
There needs to be more logic here. When you pass the footage_name to start_recording the file needs to already be created. Also, it needs to have a format. The one I saw in their docs is ".h264".